LICENSE.md

```diff
@@ -0,0 +1,15 @@
+Put your Activator template license in here, for example...
```

Comment:
Put an actual license here, without this sentence? (it also says "Copyright Typesafe" below)